<?xml version="1.0"?>
<launch>

  <rosparam command="load" file="$(find graph_test)/config/object_test.yaml"/>
  <include file="$(find cartesian_moveit_config)/launch/demo.launch"/>
  <node pkg="object_loader" name="context_manager_node" type="context_manager_node" respawn="true" output="screen"/>

  <rosparam command="load" file="$(find graph_test)/config/test_solver.yaml"/>
  <rosparam command="load" file="$(find graph_test)/config/static_scene_cartesian.yaml"/>
  <node pkg="object_loader" name="scene_spawner" type="scene_spawner" output="screen"/>

  <node name="test_solver"
        pkg="graph_test"
        type="test_solver"
        output="screen"
        launch-prefix="gdb -ex run --args" >
    </node>

</launch>
